tests/test_modules.py

```diff
@@ -101,6 +101,14 @@ def test_modules_create_succeed(self):
         module_create.create()
         assert os.path.exists(os.path.join(self.pipeline_dir, "modules", "local", "fastqc.nf"))
 
+    def test_modules_create_different_conda_name(self):
```

Comment:
This will be tested explicitly by one of the other tests I have added for the DSL2 pipeline so we can maybe remove this test.

Reply:
Fine for me, I'll remove it again 👍